easybuild/easyblocks/i/icc.py

```diff
@@ -80,23 +90,23 @@ def sanity_check_step(self):
         """Custom sanity check paths for icc."""
 
         binprefix = "bin/intel64"
-        libprefix = "lib/intel64/lib"
+        libprefix = "lib/intel64"
         if LooseVersion(self.version) >= LooseVersion("2011"):
             if LooseVersion(self.version) <= LooseVersion("2011.3.174"):
                 binprefix = "bin"
             elif LooseVersion(self.version) >= LooseVersion("2013_sp1"):
                 binprefix = "bin"
-                libprefix = "lib/intel64/lib"
             else:
-                libprefix = "compiler/lib/intel64/lib"
+                libprefix = "compiler/lib/intel64"
 
         binfiles = ["icc", "icpc"]
         if LooseVersion(self.version) < LooseVersion("2014"):
             binfiles += ["idb"]
 
         custom_paths = {
-            'files': ["%s/%s" % (binprefix, x) for x in binfiles] +
-                     ["%s%s" % (libprefix, x) for x in ["iomp5.a", "iomp5.so"]],
+            'files': [os.path.join(binprefix, x) for x in binfiles] +
+            [os.path.join(libprefix, 'lib%s' % x) for x in ['iomp5.a', 'iomp5.so']] +
+            ['include/omp.h'],
```

Comment:
this looks sensible, but there's a range of icc versions that don't have this, at least not in this location

e.g. `2011.13.367` and `2013.5.192` do *not* have this file there...

so, we should only include this conditionally (e.g. since `2015` and up)

@wpoely86: I'll fix this in a PR to your branch